### DISCUSSION TITLE
Cannot download tags with spaces in them

### DISCUSSION THREAD

**Discussion by xingpao71** [TumblThreeApp/TumblThree#559] [General]
**Describe the bug**
Cannot download links for tags with spaces in them.

**To Reproduce**
Add link for tag https://www.tumblr.com/search/Blade Runner
Add blog button is greyed out and cannot be clicked
Will only accept https://www.tumblr.com/search/Blade or https://www.tumblr.com/search/BladeRunner

**Expected behavior**
Should download all data for tag Blade Runner

**Screenshots**
![Screenshot_1](https://github.com/user-attachments/assets/84ef855a-8c71-4ad6-bda0-1d93f6347338)

**Desktop (please complete the following information):**
 - TumblThree version: 2.14.2.457
 - OS: Windows 11 Pro

**Comment by WenElevating** [comment]
Hello, I am a first-time open source project developer and I would like to study this issue. Is that okay?

**Comment by thomas694** [comment]
@xingpao71 URLs cannot contain spaces. There may be a difference between what the browser displays when you hover over a link and what the actual URL of the link is. If you copy the link of the tag (https://www.tumblr.com/tagged/blade%20runner), it works.
In fact, your URL searches for all posts that contain the term "Blade Runner" and not just those tagged with that term.

Currently the Search crawler seems to be broken, at least for this term. But the Tag Search crawler works.

@WenElevating Sure.
